pontoon/base/static/js/main.js

```diff
@@ -505,7 +512,9 @@ $(function() {
 
     listitems.sort(function(a, b) {
       return (val(index, a) < val(index, b)) ? -dir :
-        (val(index, a) > val(index, b)) ? dir : 0;
+        (val(index, a) > val(index, b)) ? dir :
```

Comment:
`val(index, a) - val(index, b)` is equivalent to this and only calls `val` twice.